src/tribler-gui/tribler_gui/widgets/lazytableview.py

```diff
@@ -161,8 +162,11 @@ def on_move_button_clicked(self, _index):
     def resizeEvent(self, _):
         if self.model() is None:
             return
+        viewport_width = self.width()
         for col_num, col in enumerate(self.model().columns):
-            self.setColumnWidth(col_num, self.model().column_width.get(col, lambda _: 110)(self.width()))
+            self.setColumnWidth(
+                col_num, col.width if col.width != EXPANDING else viewport_width - self.model().min_columns_width - 20
```

Comment:
This change would also better fit in a separate commit.

Reply:
Again, that was a necessary part of refactoring the column thing. If you'd like, I can put the channels related changes in a separate commit (in the same PR).

Reply:
Yeah making a separate commit would be appreciated 👍